### ISSUE TITLE
Multiple SAML attribute values are handled incorrectly (all but the first are omitted)

### ISSUE THREAD

**Issue by craigdrayton** [steffow/meteor-accounts-saml#26]
Thanks for this useful plugin - I was able to get started with SAML quicker than expected.

I have a SAML attribute named "groups" which returns multiple values:

```
<saml2:Attribute Name="groups" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified">

<saml2:AttributeValue xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="xs:string">Everyone</saml2:AttributeValue>

<saml2:AttributeValue xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="xs:string">Pandas</saml2:AttributeValue>

</saml2:Attribute>
```

However the package takes only the first attribute value ("Everyone") and stores it in the Meteor user profile as a string. I would expect the package to return an array of strings, in this case.

**Comment by craigdrayton** [comment]
Looks the the code responsible is here:
https://github.com/steffow/meteor-accounts-saml/blob/develop/saml_utils.js#L420

**Comment by steffow** [comment]
Correct. Still trying to fix some bugs with older PRs. Stay tuned.

**Comment by steffow** [comment]
Fixed in 0.0.15

**Comment by craigdrayton** [comment]
Thanks a lot for getting the fix in so quickly
